web/modules/custom/dpl_campaign/src/EventSubscriber/ApiRouteSubscriber.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\dpl_campaign\EventSubscriber;
+
+use Drupal\Core\Routing\RouteSubscriberBase;
+use Symfony\Component\Routing\Route;
+use Symfony\Component\Routing\RouteCollection;
+
+/**
+ * Remove X-CSRF protection for campaign match API.
+ *
+ * This reduces the complexity of calling the API as the caller no longer has
+ * to obtain a token before making the actual request.
+ */
```

Comment:
Consider to make the comment more generic. Wouldn't it be plausible that we get future routes that need to bypass the X-CSRF protection?

Reply:
We will do that some other time.